### PR TITLE
Pimp README

### PR DESCRIPTION
**Pull request by hynek**
Here's a first attempt – let's iterate.

### CODE REVIEW

**Comment by hynek** [comment]
More tagline ideas:

- "Validation for the edges, trustworthy data structures for the business."
- "Transforming chaos into elegant objects."
- "Neither square brackets, nor data validation belong into business code." 🤓

**Review by Tinche — APPROVED** [review]
I like it!

Do you think we should lean into functional composition as the main vehicle here or leave that for the docs proper?

File: README.md [hunk 1]
```diff
@@ -1,71 +1,79 @@
...
-dictionaries, lists and tuples. These data types are also the lingua franca
-of most data serialization libraries, for formats like json, msgpack, cbor,
-yaml or toml.
+---
+
+Python has a rich set of powerful, easy to use, built-in **unstructured** data types like dictionaries, lists and tuples.
+These data types effortlessly convert into common serialization formats like JSON, MessagePack, CBOR, YAML or TOML.
+
+But the data that is used by your **business logic** should be **structured** into well-defined classes, since not all combinations of field names or values are valid inputs to your programs.
+The more trust you can have into the structure of your data, the simpler your code can be, and the fewer edge cases you have to worry about.
```
[the remark this answers is not in the document]
Comment (hynek) [reply README.md:25]: Why would you remove that Oxford comma? It's correct by certain standards and it removes ambiguity. It's an enumeration of upsides.

**Comment by hynek** [comment]
> Do you think we should lean into functional composition as the main vehicle here or leave that for the docs proper?

I don't think that's a mainstream-enough topic for a README. The README should entice people to find out more, not overwhelm them with a wall of text that includes words they don't understand.

**Pull request merged**